File: classes/Empresa.php
<?php
require_once __DIR__ . '/Entidade.php';

class Empresa extends Entidade {
    private string $nome, $cnpj, $email, $status;

    public function __construct(array $d) {
        $this->id     = (int)   ($d['id']     ?? 0);
        $this->nome   = (string)($d['nome']   ?? '');
        $this->cnpj   = (string)($d['cnpj']   ?? '');
        $this->email  = (string)($d['email']  ?? '');
        $this->status = (string)($d['status'] ?? 'pendente');
    }

    public function getNome():      string { return $this->nome; }
    public function getCnpj():      string { return $this->cnpj; }
    public function getEmail():     string { return $this->email; }
    public function getStatus():    string { return $this->status; }
    public function estaAprovada(): bool   { return $this->status === 'aprovada'; }

    public function getResumo(): string {
        return $this->nome . ' [' . $this->status . ']';
    }
}

File: java/src/main/java/br/unialfa/backoffice/model/Empresa.java
package br.unialfa.backoffice.model;

public class Empresa {

    private int    id;
    private String nome;
    private String cnpj;
    private String email;
    private String status;

    public Empresa() {}

    public Empresa(int id, String nome, String cnpj, String email, String status) {
        this.id       = id;
        this.nome     = nome;
        this.cnpj     = cnpj;
        this.email    = email;
        this.status   = status;
    }

    public int     getId()       { return id; }
    public String  getNome()     { return nome; }
    public String  getCnpj()     { return cnpj; }
    public String  getEmail()    { return email; }
    public String  getStatus()   { return status; }

    public void setId(int id)              { this.id       = id; }
    public void setNome(String nome)       { this.nome     = nome; }
    public void setCnpj(String cnpj)       { this.cnpj     = cnpj; }
    public void setEmail(String email)     { this.email    = email; }
    public void setStatus(String status)   { this.status   = status; }

    @Override
    public String toString() { return nome + " (" + cnpj + ")"; }
}
